tests/test_graphql_api/test_application_event/test_filtering.py

```diff
@@ -491,6 +492,39 @@ def test_application_event__filter__by_priority__multiple(graphql):
     assert response.node(1) == {"pk": event_2.pk}
 
 
+def test_application_event__filter__schedule_priority(graphql):
+    # given:
+    # - There is an application event with schedules of different priorities
+    # - A superuser is using the system
+    schedule_high = ApplicationEventScheduleFactory.create(priority=PriorityChoice.HIGH)
+    ApplicationEventScheduleFactory.create(
+        application_event=schedule_high.application_event, priority=PriorityChoice.MEDIUM
+    )
+    schedule_low = ApplicationEventScheduleFactory.create(
+        application_event=schedule_high.application_event, priority=PriorityChoice.LOW
+    )
+    graphql.login_user_based_on_type(UserType.SUPERUSER)
+
+    # when:
+    # - User tries to filter application event schedules with a specific priority
+    query = events_query(
+        fields="pk applicationEventSchedules { pk priority }",
+        applicationEventSchedules__priority=[PriorityChoice.HIGH.value, PriorityChoice.LOW.value],
```

Comment:
Should be all snake_case (`application_svent_schedules__priority`), not sure this will work.

Reply:
Yeah, was wondering if it might mess up things that are already camelCase, but seems like that's not the case! :D